src/components/CreateGameDialog.vue

```diff
@@ -1,61 +1,62 @@
 <template>
-  <v-dialog v-model="show">
-    <template #activator="{ props }">
-      <v-btn v-bind="props" color="primary" data-cy="create-game-btn" rounded elevation="8">
+  <base-dialog v-model="show" id="create-game" title="Create Game">
```

Comment:
If we use the id prop to set the data-cy attribute then we can name this id create-game-dialog and it should have the correct selector for the test
```suggestion
  <base-dialog v-model="show" id="create-game-dialog" title="Create Game">
```